geoips/interfaces/module_based/output_checkers.py

```diff
@@ -487,7 +504,12 @@ def test_products(
         else:
             comp_str = self.name.upper() + " "
         compare_strings += [comp_str]
-        if self.module.outputs_match(self, output_product, compare_product):
+        if self.name == "image":
+            if self.module.outputs_match(
+                self, output_product, compare_product, image_compare_threshold
+            ):
+                goodcomps += [comp_str + "{0}".format(output_product)]
+        elif self.module.outputs_match(self, output_product, compare_product):
```

Comment:
I think if we went the "output_checker_kwargs" route, we would not have to have a special case for "image" family here ?  We'd just pass the kwargs in ?

Oh, hmm, this also makes me think, this could just be calling self.module(self, output_product, compare_product, **output_checker_kwargs) ?  def call calls outputs_match, so maybe we should be calling "call" here? (which would mean outputs_match is NOT actually a required method, it really just boils down to "call" as the output checker at this point?

Reply:
That comment was wrong